boards/pulga/include/sx127x_params.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2017 Inria Chile
+ * Copyright (C) 2022 Universidade de São Paulo
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_nz32-sc151
+ * @{
+ *
+ * @file
+ * @brief       sx127x configuration for nz32-sc15 using modtronix inAir9
+ *              LoRa module
+ * @author      Francisco Molina <francisco.molina@inria.cl>
+ * @author      Geovane Fedrecheski <geonnave@gmail.com>
+*/
+
+#ifndef SX127X_PARAMS_H
+#define SX127X_PARAMS_H
+
+#include "sx127x.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Board specific SX127X configuration
```

Comment:
Does the board have this radio?
Then you want to add it to `netdev_default`

Reply:
There is a shield, specifically designed for the pulga, which has the sx1272. Will check `netdev_default`.

Reply:
If I understood correctly, doing:

```Makefile
ifneq (,$(filter netdev_default,$(USEMODULE)))
  USEMODULE += sx1272
endif
```

Will automatically include the sx1272 module whenever I compile with `BOARD=pulga`.

If that is true, then I think I should not add it, since the board only has the sx1272 when the lora shield is mounted.

Reply:
Then better also add a comment here that this requires a shield to be installed, otherwise you will have people wondering why LoRa isn't working on their board :smile: